### PR TITLE
Release data 09/02/2022

### PR DESCRIPTION
**Pull request by Wkt8**
This is the test data for new imaging datasets. Additional imaging subgraphs have been added.
Specimen --> imaged_specimen --> sequence_file
specimen --> imaged_specimen --> image_file
specimen --> imaged_specimen --> analysis_file

Several entities were deleted in order to allow for our new validation in ingest (ingest-graph-validator).

Issues addressed:
https://github.com/HumanCellAtlas/metadata-schema/issues/1430

### CODE REVIEW

**Review by hannes-ucsc — DISMISSED** [review]
Something is off here. I see that two subgraphs were removed, and three added. I assume that the intent was to update two and add one.

**Comment by Wkt8** [comment]
Hi Hannes, my apologies. You should now see that 2 subgraphs remained the same, and 3 were added. This is because there are 3 subgraphs to do with imaging data and imaged_specimen. 
Specimen --> imaged_specimen --> sequence_file
specimen --> imaged_specimen --> image_file
specimen --> imaged_specimen --> analysis_file

**Review by kbergin — APPROVED** [review]

**Comment by hannes-ucsc** [comment]
@danielsotirhos will review this for me

cc: @theathorn

File: tests/metadata/donor_organism/5bc2bf2f-eb98-5510-a386-172cef38f2f2_2021-01-01T00:00:00.000000Z.json [hunk 1]
```diff
@@ -14,8 +14,6 @@
     },
     "diseases": [
         {
-            "ontology": "MONDO:0100096",
-            "ontology_label": "COVID-19",
```
Comment (ESapenaVentura) [inline tests/metadata/donor_organism/5bc2bf2f-eb98-5510-a386-172cef38f2f2_2021-01-01T00:00:00.000000Z.json]: Why are these ontologies being deleted from this entity?

File: tests/metadata/donor_organism/9173ee6a-f1b2-5762-9272-3433b5ef7530_2021-01-01T00:00:00.000000Z.json [hunk 2]
```diff
@@ -14,8 +14,6 @@
     },
     "diseases": [
         {
-            "ontology": "MONDO:0100096",
```
Comment (ESapenaVentura) [inline tests/metadata/donor_organism/9173ee6a-f1b2-5762-9272-3433b5ef7530_2021-01-01T00:00:00.000000Z.json]: Ditto to above comment

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]
Added 2 comments, other than that LGTM

File: tests/metadata/donor_organism/5bc2bf2f-eb98-5510-a386-172cef38f2f2_2021-01-01T00:00:00.000000Z.json (hunk 1 above)
Comment (Wkt8) [reply tests/metadata/donor_organism/5bc2bf2f-eb98-5510-a386-172cef38f2f2_2021-01-01T00:00:00.000000Z.json]: This is completely my bad. An error that occurred during troubleshooting some of the issues with ingesting this dataset.

**Review by dsotirho-ucsc — APPROVED** [review]
Approved.

**Review by aherbst-broad — APPROVED** [review]

**Review by ruchim — APPROVED** [review]
To be totally honest -- I'm not the best person to review what the structure of good test data looks like (just my lack of detailed knowledge on what type of cases to look out for ). I'm happy to approve because it looks right from what I know!

**Review by amnonkhen — APPROVED** [review]

**Comment by Wkt8** [comment]
This PR now has most of the reviews that were requested. @NoopDog please comment if you have any issues with this example of imaging subgraphs!

**Pull request merged**